pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/DDPInstance.java

```diff
@@ -461,6 +461,30 @@ public static List<DDPInstance> getDDPInstanceListWithKitBehavior() {
         return ddpInstances;
     }
 
+    public static DDPInstance getDDPInstanceByInstanceName(String instanceName) {
```

Comment:
It is better to move this function to `DDPInstanceDao`.

Reply:
@gmakharat good catch, actually we already have that method defined in `DDPInstanceDao`, seems that I missed it, I'll just use `DDPInstanceDao.getDDPInstanceByInstanceName` instead of this then.

Reply:
Actually it needs slight modifications, because they seem to be returning  `DDPInstance` and `DDPInstanceDto` which are different DTO classes. I'll act accordingly

Reply:
It is better to reduce usage of `DDPInstance` class to fully switch to `DDPInstanceDao`